SA3/qa_web/views.py

```diff
@@ -224,39 +276,45 @@ def vote(request):
                     post.downvotes = post.downvotes + 1
                     post.upvotes = post.upvotes - 1
         post.save()
-        return JsonResponse({'new_score' : post.score, 'id' : 'score_{}_{}'.format(post.id, post_type)})
+        return JsonResponse({'new_score': post.score, 'id': 'score_{}_{}'.format(post.id, post_type)})
     # Accessing url without using Post or unauthenticated
     return HttpResponseRedirect('/')
 
 
-# Home Page
 def homepage(request):
+    """Displays the website's home page.
```

Comment:
This comment in particular seems to do nothing other than state the obvious (the function name makes this function pretty self explanatory).